CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing to Talawa API
+Thank you for your interest in contributing to Talawa API. Regardless of the size of the contribution you make, all contributions are welcome and are appreciated. 
+
+If you are new to contributing to open source, please read the Open Source Guides on [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/).
+
+## Ways to Contribute
+If you are ready to start contributing code right away, we have a list of [good first issues](https://github.com/PalisadoesFoundation/talawa-api/labels/good%20first%20issue) that contain issues with a limited scope. 
+
+## Quicklinks
+
+* [Our Development Process](#Our-development-process)
+    * [Issues](#issues)
+    * [Pull Requests](#pull-requests)
+    * [Git Flow](#git-flow)
+* [Contributing Code](#contributing-code)
+* [GSoC](#gsoc)
+* [Community](#community)
+
+
+
+### Our Development Process
+We utilize GitHub issues and pull requests to keep track of issues and contributions from the community. 
+
+#### Issues 
+Feel free to open an issue if you spot any bugs or improvements you see in Talawa. Once an issue has been created, it will be triaged by the members of the Palisadoes Foundation. If passed, the issue will be open for assignment. 
+
+As you work through these issues, you will start to get a better understanding of the type of problems that get reported. You may even start noticing the same issue gets reported. In these cases, you can close the issue and add a comment that says “Duplicate of #issue”.
+
+#### Pull Requests
+Every pull request should be linked to an existing issue. A pull request should consist of a summary, to help us understand the motivation behind the changes. 
+
+#### Git Flow
+For Talawa API, we utilize the GitFlow branching model. GitFlow is geared towards efficiently tracking development and managing releases. The model makes parallel development efforts easy and safe by isolating new development efforts from completed work. 
+
+The different types of branches we may use are:
+* Feature branches (feature/branch-name)
+* Release branches (release/1.XX)
+* Bug branches  (bugfix/branch-name)
+* Hotfix branches (hotfix/branch-name)
+
+Detailed document containing how GitFlow works: https://nvie.com/posts/a-successful-git-branching-model/
+
+### Contributing Code
+Code contributions to Talawa come in the form of pull requests. These are done by forking the repo and making changes locally. 
+
+The process of proposing a change to Talawa can be summarized as:
```

Comment:
LGTM. I would like to see more detailed instructions(commands to use) here to follow by new contributors.

Reply:
I guess a link to [configuration.md](https://github.com/PalisadoesFoundation/talawa-api/blob/master/configuration.md) would do the work, without increasing the size of CONTRIBUTION.md

Reply:
@utkarshshendge, Let me explain it again. I was asking you to add detailed instructions in contributing code section - command to clone repo, add/update new files, commit changes and then push it to remote branch.